R/load-sherlock.R

```diff
@@ -221,28 +221,30 @@ add_genetic_identification <- function(con, sample_identifiers) {
   ) |>
     dplyr::mutate(
       status_code_id = dplyr::case_when(
```

Comment:
let's go through this one more time together and add some comments to help guide interpretation